src/definition-finder.ts

```diff
@@ -0,0 +1,49 @@
+'use babel';
+
+import MouseMoveListener from './mouse-move-listener';
+import {SourceRange} from 'polymer-analyzer/lib/model/model'
+
+class DefinitionFinder extends MouseMoveListener {
```

Comment:
Hm, I think I've given this feedback the last time we talked about it. It seems like Atom doesn't support this out of the box.

https://atom.io/packages/hyperclick looks like the closest thing but it seems to be for providing documentation popups!? Arg.

Ok, this is just me ranting, plz ignore.